compiler/src/iree/compiler/Dialect/Flow/IR/test/tensor_ops.mlir

```diff
@@ -215,12 +215,12 @@ util.func public @tensorEncodeDynamic(%arg0 : tensor<?x4xf32>, %arg1 : index, %a
 // CHECK-LABEL: @tensorEncodeChangeEncoding
 // CHECK-SAME:    %[[ARG0:.[a-zA-Z0-9]+]]
 // CHECK-SAME:    %[[ARG1:.[a-zA-Z0-9]+]]
-#encoding = #iree_encoding.unspecialized_encoding<123>
-#encoding1 = #iree_encoding.unspecialized_encoding<456>
+#encoding = #iree_encoding.specialized<123>
```

Comment:
Didn't you rename #iree_encoding.unspecialized_encoding to #iree_encoding.specialization_resolver?

Reply:
It's a misuse. I identified it when I did the replacements. Let me reflect the change in the PR description, I missed it. (I only had a separate commit mentioned it.)